src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/Namespaces.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.NetCore.Analyzers.Runtime
+{
+    using System.Collections.Generic;
+
+    internal static class Namespaces
+    {
+        internal static readonly IReadOnlyList<string> System = new[]
```

Comment:
Can you double check that all members in the class are actually used? 1 quick way to check in VS in the number of references shows above the symbol definitions. It if shows 0, you can delete it. I'm pretty certain you'll get a warning if there are no references to a member though, so I'm not too worried about this